server/server.go

```diff
@@ -115,7 +115,8 @@ func NewServer(ctx *Context) (*Server, error) {
 
 	s.kvDB = kv.NewDBServer(sender)
 	s.kvREST = kv.NewRESTServer(s.kv)
-	s.node = NewNode(s.kv, s.gossip)
+	// TODO(bdarnell): make StoreConfig configurable.
```

Comment:
Do you have anything in mind for this? I'd just remove the TODO unless there's a good reason we're going to want to change default values in production (if yes, then the StoreConfig should perhaps be part of the server.Context).